### DISCUSSION TITLE
Please, bundle together emails from failed github actions.

### DISCUSSION THREAD

**Discussion by codesnik** [community/community#6054] [Actions]
We have quite an extensive set of workflows in some of our repositroes. When they're failing, or being canceled after subsequent push, my inbox is getting bombarded by emails. But unlike, say, notifications about PR comments, those aren't bundled in threads in any email client I know.

Suggestion: please include some kind of github object reference in notification's email References or In-Reply-To headers, the same way as usual PR discussion. Probably the best would be to have them grouped by git-SHA or some other common id per triggering event, if there's any.

**Comment by thenextman** [comment]
Likewise, when running a job referencing an environment that requires review: if the job is run in a matrix, we get an individual email for each matrix item. It would be nice to coalesce these emails.
